dashboard/test/factories/user_level_infos.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: user_level_infos
+#
+#  id            :integer          not null, primary key
+#  time_spent    :integer          default(0)
+#  user_level_id :integer          unsigned
+#
+
+FactoryGirl.define do
+  factory :user_level_info do
+    time_spent 10
+    user_level_id 1
+  end
```

Comment:
our convention is to put factory definitions in factories.rb, unless they are factories for models which are in a different namespace . please move these 4 lines into that file, and then delete this file. You could put it near here: https://github.com/code-dot-org/code-dot-org/blob/2866ed188114c4f67ba39e94ad7b533e60710458/dashboard/test/factories/factories.rb#L804

As long as your new unit test is still passing locally after that, you should be good to merge and you can `[ci skip]` to skip drone (or just not wait for drone) if you want.